doc/nrf/releases_and_maturity/releases/release-notes-3.1.0.rst

```diff
@@ -0,0 +1,1183 @@
+
+    This change in the nRF54L10 partition map is a breaking change and cannot be performed using DFU.
+    As a result, the DFU procedure will fail if you attempt to upgrade the sample firmware based on one of the |NCS| v3.0 releases.
+  * The behavior of the :ref:`nrf_desktop_usb_state_pm` on USB cable disconnection.
+    While disconnecting the USB cable, the :c:enumerator:`USB_STATE_SUSPENDED` USB state might be reported before the :c:enumerator:`USB_STATE_DISCONNECTED` USB state.
+    For application to behave consistently regardless of whether the :c:enumerator:`USB_STATE_SUSPENDED` USB state was reported, the module submits a :c:struct:`force_power_down_event` to force a quick power down.
+    The module also restricts the power down level to the :c:enumerator:`POWER_MANAGER_LEVEL_SUSPENDED`.
+    Then, after the delay configurable by :ref:`CONFIG_DESKTOP_USB_PM_RESTRICT_REMOVE_DELAY_MS <config_desktop_app_options>` Kconfig option, the module removes the power down level restriction.
+    This allows you to take actions, such as restarting Bluetooth LE advertising, after disconnecting the USB cable without going through reboot.
+  * The configurations for nRF54L-based board targets that store the MCUboot verification key in the KMU peripheral to automatically generate the :file:`keyfile.json` file in the build directory (the :kconfig:option:`SB_CONFIG_MCUBOOT_GENERATE_DEFAULT_KMU_KEYFILE` sysbuild Kconfig option) based on the input file provided by the :kconfig:option:`SB_CONFIG_BOOT_SIGNATURE_KEY_FILE` sysbuild Kconfig option.
+    This KMU provisioning step can now be performed automatically by the west runner, provided that a :file:`keyfile.json` file is present in the build directory.
+    The provisioning is only performed if the ``west flash`` command is executed with the ``--erase``  or ``--recover`` flag.
+
+nRF Machine Learning (Edge Impulse)
+-----------------------------------
+
+* Added:
+
+  * Power-optimized configuration for the :zephyr:board:`nrf54h20dk` board target.
+  * A single-core release configuration for the :zephyr:board:`nrf54h20dk` board target.
+
+Serial LTE modem
+----------------
+
+* Added:
+
+  * The ``AT#XAPOLL`` command to asynchronously poll sockets for data.
+  * The send flags for ``#XSEND``, ``#XSENDTO``, ``#XTCPSEND`` and ``#XUDPSEND`` commands.
+  * The send flag value ``512`` for waiting for acknowledgment of the sent data.
+
+* Updated:
+
+  * The ``AT#XPPP`` command to support the CID parameter to specify the PDN connection used for PPP.
+  * The ``#XPPP`` notification to include the CID of the PDN connection used for PPP.
+  * The initialization of the application to ignore a failure in nRF Cloud module initialization.
+    This occurs sometimes especially during development.
+  * The initialization of the application to send "INIT ERROR" over to UART and show clear error log to indicate that the application is not operational in case of failing initialization.
```

Comment:
```suggestion
  * The initialization of the application to send "INIT ERROR" over UART and show clear error log to indicate that the application is not operational in case of failing initialization.
```